peps/pep-0661.rst

```diff
@@ -296,18 +272,14 @@ simplified version follows::
 
             sentinel = super().__new__(cls)
             sentinel._name = name
-            sentinel._repr = repr
-            sentinel._bool_value = bool_value
             sentinel._module_name = module_name
+            sentinel._repr = name.split(".")[-1]
```

Comment:
This made me think, should there be special handling for e.g. `Sentinel("MyClass.INNER")`, if the `_repr` splits on `.`?

Reply:
What kind of "special handling" do you have in mind?

Reply:
I'm not entirely sure. One option would be raise an error if `name` contained a full stop. Another would just have `_repr = name`. I suppose the parallel I'm thinking of is:

```pycon
>>> class MyClass:
>>>     class SentinelClass: ...
>>>
>>>     def __init(self): pass
>>> MyClass.SentinelClass
__main__.MyClass.SentinelClass
```

Reply:
I might consider using the full name provided as the repr. I'll think about it.

Reply:
Maybe use `str.rpartition('.')` instead?

Reply:
That would indeed work well, though that is a less well-known method so I think I'll avoid it for this simplified reference implementation.

Reply:
Actually, I've opted to use the full name for the repr in such cases, so this is no longer needed.